zeppelin-interpreter/src/test/java/org/apache/zeppelin/cluster/ClusterSingleNodeTest.java

```diff
@@ -20,21 +20,22 @@
 import org.apache.zeppelin.cluster.meta.ClusterMetaType;
 import org.apache.zeppelin.conf.ZeppelinConfiguration;
 import org.apache.zeppelin.interpreter.remote.RemoteInterpreterUtils;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
 
-public class ClusterSingleNodeTest {
```

Comment:
`public` before `class` should not be removed

Reply:
I use SonarLint in my IDE and it complains when `public` is present.
The following rule is raised. [JUnit5 test classes and methods should have default package visibility](https://rules.sonarsource.com/java/RSPEC-5786) 

As you can read in the [linked documentation](https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods) about JUnit5, `public` should be omitted.

> It is generally recommended to omit the public modifier for test classes, test methods, and lifecycle methods unless there is a technical reason for doing so

Reply:
I see. Thanks for explaining。